src/main/java/com/puppycrawl/tools/checkstyle/api/AutomaticBean.java

```diff
@@ -64,6 +65,7 @@
      *
      * @return a configured BeanUtilsBean
      */
+    // -@cs[ExecutableStatementCount] No other way around adding these beans.
```

Comment:
I think we can optimize this method (registration of integral types is typical, could be moved to general method) and split it into two methods (geristerIntegraltypes , registerCustomTypes)